docs/source/tutorial/using_wandb.rst

```diff
@@ -0,0 +1,3 @@
+Using Weights and Biases
+========================
+`Weights and Biases <http://wandb.ai/>`_ is a...
```

Comment:
@migalkin @mali-git here's where the tutorial should go

Reply:
Done in 37f61546fbeb5ac150dc923b69e733a2ed1b4574